Tests/CloudFoundryEnvTests/MainTests.swift

```diff
@@ -27,6 +27,10 @@ import Configuration
 
 @testable import CloudFoundryEnv
 
+let configFileURL: URL = URL(fileURLWithPath: #file).appendingPathComponent("../resources/config.json").standardized
+let configFilePath = "../resources/config.json"
+let currentPath = #file
```

Comment:
https://bugs.swift.org/browse/SR-4325 - This block was moved up because of a 3.1 bug.